test-cases/tests/alerts/c03-verify-that-alerting-mechanism-works.md

```diff
@@ -2,6 +2,9 @@
 estimate: 30m
 tags:
   - happy-path
+  - automation-jira-created
```

Comment:
@jhellar I think that if we have the `automation_jiras` we don't need also a tag, we can modify the tool to automatically add the tag if it finds an `automation_jiras`

Reply:
ok, should be fixed now